### ISSUE TITLE
Page crashed: regeneratorRuntime is not defined

### ISSUE THREAD

**Issue by MajidSadrr** [typesense/docusaurus-theme-search-typesense#15]
## Description

I'm using typesense in my docusaurus docs. I couldn't figure out why this error happened, but my search bar is not working and I have this error:
![image](https://user-images.githubusercontent.com/76779095/169977291-8c63ab03-698b-417f-af81-de3e4f774439.png)

Any ideas about the problem and how to fix that?

**Comment by jasonbosco** [comment]
Could you try installing `@babel/runtime`? It's usually not needed with docusaurus, so I wonder why you're seeing this error...

**Comment by jmccann** [comment]
I just updated to docusaurus beta.20 and started seeing this too

**Comment by jmccann** [comment]
I am running `2.0.0-beta.17` with no issues (using `0.4.0-2`).

But if I:

```
rm yarn.lock
yarn install
```

And then try again it breaks.  This is while being pinned to `2.0.0-beta.17`. So seems some other dep updates that causes this to break.

I also tried using my currently `yarn.lock` and updating to `2.0.0-beta.18` and also ran into this issue.

**Comment by MajidSadrr** [comment]
Thanks @jmccann,
I've downgraded docusaurus version to `2.0.0-beta.17` and it fixed.

**Comment by jmccann** [comment]
I don't think that is the fix ... probably need to fix this for `beta.20` as well ... so may want to re-open the issue.

**Comment by jasonbosco** [comment]
Could you check if you see the same issue with docusaurus `beta.21` and `0.4.0` of `docusaurus-theme-search-typesense`?

**Comment by jmccann** [comment]
Seems better so far with `beta.21` and `0.4.0` 👍

**Comment by MajidSadrr** [comment]
Hey,
I'm getting this issue again after upgrading docusaurus to `v2.0.0-rc.1` and typesense to `v0.5.0`. Am I doing sth wrong?

**Comment by jasonbosco** [comment]
@MajidSadrr Thank you for catching that. This was a bug I've fixed in `v0.5.1` of `docusaurus-theme-search-typesense`. Could you try that version out?

**Comment by MajidSadrr** [comment]
@jasonbosco Solved. Thank you so much!
